### DISCUSSION TITLE
Form Layout Examples

### DISCUSSION THREAD

**Discussion by AlexHedley** [microsoft/fluentui-blazor#3080] [Q&A] [answered]
Has anyone got any examples of _Form_ layouts?

Would you suggest to use something like Bootstrap etc or is it better to use `FluentGrid` / `FluentGridItem`?

- https://www.fluentui-blazor.net/Grid

## Alternatives

**Bootstrap**

![image](https://github.com/user-attachments/assets/722df10e-0505-4988-b149-6705c421eb53)

- https://getbootstrap.com/docs/5.3/forms/layout/

**Comment by AlexHedley** [comment]
Is this deployed anywhere?

**New Ship Entry Form**

- https://github.com/microsoft/fluentui-blazor/blob/main/examples/Demo/Shared/Pages/BasicFormBlazorComponents.razor
- https://github.com/microsoft/fluentui-blazor/blob/main/examples/Demo/Shared/Pages/BasicFormFluentUIComponents.razor

**Reply by vnbaaij** [reply, marked as answer]
I believe on the form overview page (the Fluent variation)

**Reply by AlexHedley** [reply]
Thanks.

**Fluent UI Blazor forms overview**

_Example_

**Basic Fluent UI form**

https://www.fluentui-blazor.net/Forms
